app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topic-controller");
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, request, response, next) => {
```

Comment:
Really good work here
As your error handling starts to grow (trust me, it will!) 
think about extracting this logic into its own file and requiring the functions into your app.js just like we do with the controllers.